### ISSUE TITLE
Parameter GUI not found false warning if only PARAMETER_GUI_DIR is set

### ISSUE THREAD

**Issue by arijitlaik** [geodynamics/aspect#1890]
the warning Parameter GUI not found: install and provide a hint using.... pops up if PARAMETER_GUI_DIR, utterly trivial issue, fixit may be
```
arijit@arijit-X201E:~/bin/aspect$ cmake .
-- ====================================================
-- ============ Configuring ASPECT ====================
-- ====================================================
-- Found deal.II version 8.5.0 at '/home/arijit/bin/deal.II-v8.5.0/lib/cmake/deal.II'
-- Using ASPECT_USE_PETSC = 'OFF'
-- Setting up test project, see tests/setup_tests.log for details.
-- Enabling dynamic loading of plugins from the input file
-- Enabling checking of compatible deal.II library when loading plugins
-- Autopilot invoked
-- Run   $ make info  to print a detailed help message
-- Runtime floating point checks enabled.
-- Disabling cotire.
-- Parameter GUI not found: install and provide a hint using -D PARAMETER_GUI_DIR or set -D PARAMETER_GUI_EXECUTABLE directly.
-- Writing config into detailed.log...
-- Configuring done
-- Generating done
-- Build files have been written to: /home/arijit/bin/aspect
arijit@arijit-X201E:~/bin/aspect$ echo $PARAMETER_GUI_DIR 
/home/arijit/bin/parameter_gui
```

**Comment by gassmoeller** [comment]
Hi @arijitlaik, thanks for raising this issue. 
Unfortunately, I was not able to reproduce your problem, could you clarify this a bit for me: Is `/home/arijit/bin/parameter_gui` a directory containing the parameter GUI, or is that the path of the executable (or a link to the executable)? cmake should look for an executable in the directory `$PARAMETER_GUI_DIR` so if `/home/arijit/bin/parameter_gui` is the executable the correct path for `$PARAMETER_GUI_DIR` would be `/home/arijit/bin`. 
In case it is the directory of the parameter gui, is the path to your executable `$PARAMETER_GUI_DIR/parameter_gui` or is it `$PARAMETER_GUI_DIR/bin/parameter_gui`? cmake will currently only pick up the first version, but this is something we could easily change.

**Comment by arijitlaik** [comment]
/home/arijit/bin/parameter_gui is the directory containing the source for the parameter GUI and is the value for $PARAMETER_GUI_DIR, anyway i guess, i way reading the installation maual and thought that setting the PARAMETER_GUI_DIR environment variable to point to the source(and where its build) for parameter_gui was required. anyway  /home/arijit/bin/parameter_gui is the build directory and /home/arijit/bin/parameter_gui/bin/ is where i have e parameter_gui. Its a problem with my understanding sorry about that. i might have to rebuild aspect again with the first option.

**Comment by gassmoeller** [comment]
No harm done, glad to be of help. Still we should make cmake more forgiving with this kind of search. I opened #1893 that should prevent these misunderstandings. 
Could you do me a favor and after #1893 is merged try your initial approach of setting `PARAMETER_GUI_DIR=/home/arijit/bin/parameter_gui` and report back if that works?

**Comment by arijitlaik** [comment]
well, i got a bit late replying back, the PR is merged into the master and things works. Thanks :)

**Comment by gassmoeller** [comment]
Ok, good to know. Let us know if you encounter other problems.

**Comment by arijitlaik** [comment]
Totally Unrelated Question.
I would like to work on some  thin skinned tectonic models of fold thrust belts, i was wondering how can one include basal friction and or contact friction parameters in aspect, i have seen @anne-glerum paper but there basal friction is not taken into account, Can i put this up as an issue for a proposal and or new feature.

**Comment by tjhei** [comment]
> Totally Unrelated Question.

Correct. Please direct questions like this to the mailing list.
